src/components/profile/userDetailsForm/userImage/userImage.tsx

```diff
@@ -0,0 +1,70 @@
+import loadingAnimation from "images/loading.svg";
+
+import { useEffect, useRef, useState } from "react";
+
+import { updateImage } from "@/api/apiUser";
+import Form from "@/elements/form/form";
+import Button from "@/elements/button/button";
+import styles from "./userImage.module.scss";
+
+export default function UserImage({ profileImg }: { profileImg?: string }) {
+  const [showLoading, setShowLoading] = useState(true);
+  const [uploadedImg, setUploadedImg] = useState<string>();
+  const fileInputRef = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
+    const timeoutId = setTimeout(() => {
+      setShowLoading(false);
+    }, 1500);
```

Comment:
It's bad practice. The user shouldn't wait if the server processes requests very fast. Moreover images download very fast in 99%. I recommend to remove it. Images must be downloaded without spinners